content/en/infrastructure/containers/orchestrator_explorer.md

```diff
@@ -140,6 +140,219 @@ All of these columns support sorting, which helps you to pinpoint individual wor
 
 {{< img src="infrastructure/livecontainers/orch_ex_resource_utilization_sorted_column.png" alt="Container Resource Utilization Sorted Columns" style="width:50%;">}}
 
+## Query Filter Details
+
+You can narrow down the displayed resources by supplying a query within the "Filter by" search bar on the top left of the page.
+
+### Syntax
+
+A query filter is composed of terms and operators.
```

Comment:
It might be helpful for a user to have visual understanding of these concepts, as you mention these many times in the rest of the doc

Something simple, like

<img width="654" alt="image" src="https://github.com/DataDog/documentation/assets/1636933/476c6503-9804-4a45-a212-30e10d2d52ee">

Reply:
great idea :)